rudder-core/src/main/resources/hooks.d/readme.adoc

```diff
@@ -0,0 +1,32 @@
+= Generalities about hooks
+
+*Hooks are organized in subdirectories*. The root of the sub-directories is ```/opt/rudder/etc/hooks.d/```
+
+Each sub-directory has a name related to the event that will trigger the hooks
+execution.
+By default, a hook directory contains a template example and a Readme.txt file
+explaining the generalities about hooks and the specificities of that hook (parameters, etc).
+
+*Hooks must be executable*. All executable file will be used as hooks, and
+non executable one will be ignored (which allows to put other files in these
+directories, like a readme, for example).
+
+*Hooks parameter are passed by environment variable*. Rudder will fill dedicated
+environment variable for each hooks.
+
+*Hooks are executed sequentially, in lexical order*. We encourage
+you to use the patter "NN-hookname", with NN a number like
+"05", "20", etc.
+
+*Hooks have normalized returned code*. Return codes on hooks are interpreted as follow:
+
+- ```0     ```: success, no log (appart if debug one)          , continue to next hook
+- ```1-31  ```: error  , error   log in /var/log/rudder/webapp/, stop processing
+- ```32-63 ```: warning, warning log in /var/log/rudder/webapp/, continue to next hook
+- ```64-255```: reserved for future use case. Behavior may change without notice.
+
+
+== Available Hooked events
```

Comment:
We should avoid this level of title, as we are limited to 5 in the doc. We should use == here for different hook types.